### ISSUE TITLE
Install requires libzstd-dev

### ISSUE THREAD

**Issue by rawslaw** [wiedehopf/adsbexchange-978#1]
I'm running Piaware image 1090&978 with adsbexchange standard (only 1090).  Following your script attempting to add 978 feed, compilation error on install.

When running the install script on Pi 3B+, image piaware 5.10.60-v7+ on buster:
```
In file included from readsb.h:101,
                 from interactive.c:54:
threadpool.h:36:14: fatal error: zstd.h: No such file or directory
     #include <zstd.h>
              ^~~~~~~~
compilation terminated.
```

Added `libzstd-dev` to the required `$packages` in `install.sh` and installation succeeded.

**Comment by wiedehopf** [comment]
Yeah, this package isn't really needed anymore, the normal feed scripts should send 978 as well now.
But will still fix.

**Comment by rawslaw** [comment]
Interesting, I will check that out.  I recently noticed that my 978 traffic was not reflected on ADSBx, hence looked for a simple addition.  I did see that some scripts combine 1090 & 978 into dump1090 and that's not what I want.  Since I'm running 1090-fa and 978-fa, I thought I needed a separate task.

Still tinkering a bit, as I want ADSBx to pick up location from gpsd, not a file.  Thank you!

**Comment by wiedehopf** [comment]
just update the main feed scripts.

for MLAT changing location is annoying.
so yeah i'm not sure if i ever will get around to implementing gpsd support.

**Comment by rawslaw** [comment]
> for MLAT changing location is annoying.

Yeah, I've disabled the mlat service for this reason and it also doesn't apply to my use case.

I see your combined solution at ADSBexchange/feedclient@a88d549; I'll work on integrating into my setup.  Seems when I set this up a few months ago, I followed your [advice on reddit](https://www.reddit.com/r/ADSBexchange/comments/tsze6n/comment/i4rg509/?utm_source=share&utm_medium=web2x&context=3) for feeding gpsd into readsb/tar1090. 

I agree, probably not worth adding gpsd support for normal users who set up a feeder. Mine is an edge case and I have the know-how to tinker with bash scripts and service files.  

I'm primarily using this feeder to sample if additional coverage is beneficial in various remote or low altitude locations, like airport surface and over water where more sites would pick up traffic lower to the ground.  While I'm gathering this data for myself / clients, it is valuable to share it with the community as often I capture traffic that is otherwise uncollected.   

Thank you for your work.
